RetailCoder.VBE/UI/SourceControl/SourceControlViewViewModel.cs

```diff
@@ -63,6 +69,14 @@ public class SourceControlViewViewModel : ViewModelBase
             ListenForErrors();
         }
 
+        private void _state_StateChanged(object sender, ParserStateEventArgs e)
+        {
+            if (e.State == ParserState.Parsed)
```

Comment:
Should maybe also disable interactions when there's a StateChange into anything else, to prevent commiting code that's not parsing. That's all assuming that the parser actually parses everything that compiles...